co-log-core/co-log-core.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       2.4
 name:                co-log-core
-version:             0.2.0.0
+version:             0.2.0.1
```

Comment:
No need to bump up version here. The package can be revised on Hackage if only dependencies bump ups are required without code changes.

Reply:
Alright, removed that change :)